Add new mission where the player tries to get an Unfettered translator (#6515)
endless-sky/endless-sky: data/wanderer/wanderers start.txt

	source
		attributes "wanderer"
	to offer

> 		has "discovered hai"

Reviewer: Technically the player could have "discovered" hai space but not actually gone through the first contact, as it's possible to just keep deferring the first contact missions. You should instead check if either of the first contact missions have been finished.
```suggestion
		or
			has "First Contact: Hai: offered"
			has "First Contact: Unfettered: offered"
```

		sprite "label/wanderers"

mission "Wanderers: Unfettered Diplomat"
	landing
	source
		attributes "unfettered"
	to offer
		has "First Contact: Wanderer: offered"
		has "First Contact: Unfettered: offered"
		not "unfettered know of wanderers"
		not "Wanderers: Hai Diplomat: offered"
		not "Wanderers: Diplomacy: offered"
	on offer
		conversation
			`The aliens north of Hai space seem to speak the Hai language. Do you want to look for someone here who could serve as an interpreter?`
			branch hai
				has "First Contact: Hai"
			choice
				`	(Yes.)`
					goto try
				`	(Not right now.)`
					defer
			label hai
			choice
				`	(Yes.)`

> 				`	(No; the Fettered Hai are better suited for this.)`

Reviewer 2: ```suggestion
				`	(No; the Fettered Hai are better suited for this.)`
```

I'd suggest reworking this bit. Something like "the peaceful Hai" or the "Hai central government."

"Fettered" is not the official name, and if choosing this option, player is probably not fully sharing in the Unfettered bias of using that phrase.

Author: Changed to "the Hai southern of here"

> 				`	(No; the Hai southern of here are better suited for this.)`

Reviewer 2: ```suggestion
				`	(No; the other Hai south of here are better suited for this.)`
```

For this phrasing, probably best to go this way then. :)

					decline
			label try

> 			`	You search around the spaceport for any sign of a local government. Eventually, one of the Hai approaches you, and she says, "You are lost, monkey. What is it that you are looking for?"`

Reviewer 3: ```suggestion
			`	You search around the spaceport for any sign of a local government. Eventually, one of the Hai approaches you. "You are lost, monkey. What is it that you are looking for?"`
```

			choice
				`	"Do you know where I can talk to an official?"`
				`	"Do you know where I could find an interpreter?"`
					goto interpreter
				`	"I'm doing fine, thank you."`
					goto decline
			`	The Hai says, "If your reason for talking to our chiefs is worthwhile, then I will take you to them."`
			`	"I'm looking for a Hai interpreter," you say.`
			label interpreter

> 			`	She looks back at you quizically. "What purpose would an interpreter serve? You can understand my words, can you not?"`

Reviewer 3: ```suggestion
			`	She looks back at you quizzically. "What purpose would an interpreter serve? You can understand my words, can you not?"`
```

			choice
				`	"I need one to talk to the aliens up north from here."`
				`	"I think that I'll be better off finding one of the chiefs on my own."`
					goto decline
			`	The Hai scrunches up her nose. "Foolish monkey. We have already contacted those aliens; we have little use for another emissary."`
			choice
				`	"Can you still provide an interpreter, though?"`
				`	"I understand."`

> 					label end

Reviewer 4: you typed label end instead of goto end here

> 			`	"You do not need an interpreter. All you need to know is that those aliens are not worth conversing with."`

Reviewer 2: ```suggestion
			`	"You do not need an interpreter. All you need is to know that those aliens are not worth conversing with."`
```

Since you're not asking to know anything, per se.
